internal/controller/operatorconfigmap_controller.go

```diff
@@ -64,11 +64,12 @@ const (
 	operatorConfigMapName = "ocs-client-operator-config"
 	// ClusterVersionName is the name of the ClusterVersion object in the
 	// openshift cluster.
-	clusterVersionName     = "version"
-	manageNoobaaSubKey     = "manageNoobaaSubscription"
-	subscriptionLabelKey   = "managed-by"
-	subscriptionLabelValue = "webhook.subscription.ocs.openshift.io"
-	generateRbdOMapInfoKey = "generateRbdOMapInfo"
+	clusterVersionName          = "version"
+	manageNoobaaSubKey          = "manageNoobaaSubscription"
+	subscriptionLabelKey        = "managed-by"
+	subscriptionLabelValue      = "webhook.subscription.ocs.openshift.io"
+	generateRbdOMapInfoKey      = "generateRbdOMapInfo"
+	csiCtrlPluginHostNetworkKey = "useHostNetworkForCsiControllers"
```

Comment:
Please have the var name match the value
```suggestion
	useHostNetworkForCsiControllersKey = "useHostNetworkForCsiControllers"
```